src/cli/npc.rs

```diff
@@ -193,7 +200,7 @@ impl CliIfaceBrief {
                             ));
                         }
                         if let Some(fwd) = ip_info.forwarding {
-                            addr_strs.push(format!("forwarding {}", fwd));
+                            addr_strs.push(format!("forwarding {fwd}"));
                         }
```

Comment:
This seems to be a unrelated change.

Reply:
There seem to be several other unrelated changes in this commit like this.

Reply:
The second commit is for cargo clippy and cargo fmt.

Reply:
The first commit also contains lot of cargo clippy/fmt changes, making the review more difficult. In following PRs please try to split and structure in logical and small commits to facilitate reviews.